survey/docker.py

```diff
@@ -43,7 +43,10 @@ async def __aenter__(self):
     async def __aexit__(self, exc_type, exc_value, traceback):
         # The Python documentation says communicate always closes stdin,
         # but it only closes if a value is passed.
-        await self.process.communicate(b"")
+        try:
+            await self.process.communicate(b"")
+        except Exception:
```

Comment:
It is normally not a good idea to catch a bare `Exception`, but it looks like the intention here, inside an `__aexit__`, is to ensure the process is not hung up for any reason.